generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -18,6 +19,7 @@ public CheckpointOperation(State state) {
     @Override
     public void execute() {
         try {
+//            Correctness.recordOperation("Checkpoint, start", false);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)